src/pages/Receive.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <mobile-page
+    :redirect-to-on-close="{ name: 'transfer' }"
+    class="receive"
+    fill="neutral"
+    back-button
+  >
+    <guide
+      fill="primary"
+    >
+      <em>Receive</em> tokens
+      <div class="note">
+        Scan this QR code to receive
+        <br>tokens on this address
+      </div>
+    </guide>
+
+    <ae-account-reverse v-bind="activeIdentity" />
+
+    <template slot="content-bottom">
```

Comment:
template can be omitted by adding `slot="content-bottom"` to child ListItem component